spring-grpc-core/src/main/java/org/springframework/grpc/server/ServerBuilderCustomizer.java

```diff
@@ -15,11 +15,20 @@
  */
 package org.springframework.grpc.server;
 
-import java.util.function.Consumer;
-
 import io.grpc.ServerBuilder;
 
+/**
+ * Callback interface that can be used to customize a {@link ServerBuilder}.
+ *
+ * @author Chris Bono
+ */
 @FunctionalInterface
-public interface GrpcServerConfigurer extends Consumer<ServerBuilder<?>> {
+public interface ServerBuilderCustomizer {
```

Comment:
Renamed to "customizer" rather than "configurer" to match Boot. 

Motivation is that "configurer" is typically singular in boot and it does a "provide a <thing> with sensible defaults" and then `0...N` "customizer" tweak that initial "configure" of the thing.